src/main/resources/application.properties

```diff
@@ -12,4 +12,5 @@ log.terraform.stdout.stderr=true
 terraform.binary.location=
 terraform.log.level=INFO
 otel.exporter.otlp.enabled=false
-terraform.root.module.directory=
\ No newline at end of file
+terraform.root.module.directory=
+deployment.clean.workspace.enabled=true
```

Comment:
we can call the property -` clean.workspace.after.deployment`.

Please also add this to the properties table in the readme file.

Reply:
Updated.